integration_tests/pages/recentArrivals/prisonerSummary.ts

```diff
@@ -0,0 +1,14 @@
+import Page, { PageElement } from '../page'
+
+export default class PrisonerSummaryPage extends Page {
+  constructor(title, hasBackLink, hasFeedbackBanner) {
+    super(title, hasBackLink, hasFeedbackBanner)
```

Comment:
you don't need to have parameters for `hasBackLink, hasFeedbackBanner` as they don't change?

Reply:
These were added as the summary page doesn't have a backlink yet and page.ts is set to true as default
`    private readonly backNavigationLink: BackLink = { hasBackLink: true },`